dopeshot/truth-or-dare-frontend
# Collaboration templates

Example issue based on template:

## Summary
Add delete button for users

## Problem

###  Explanation
As of now users have no way of deleting their account. The backend however provides this functionality. Therefore a button should be added that creates a way for the user to delete their accounts. This is not also a UX requirement but also necessary from a legal standpoint.

### Use case
User that wants to delete their account

## Benefits
Closer to a (legally correct) version of the app that can be released 

# Possible Solution
It would only be logical to add this button in the settings and it should only be visible for logged in users.
to ensure a consistent and understandable UX adding the button in red and providing a confirmation modal.

# Resources
- [tutorial video](https://www.youtube.com/watch?v=5DlROhT8NgU)

#Example PR based on template:

# Changes
## List
- add confirmation modal component
- add account delete button in settings
- add confirmation modal for account delete

## Before and after
(Optional)
(Provide before and after screenshots to make it easier for the reviewer to identfy changes)

# Checklist

## Tests
- [x] cypress tests exist

## Clean Code
- [x] comments exist
- [x] no errors in browser console

# Setup
delete node modules and run npm install as one package has been downgraded

# Review steps
1. create sample account
2. login to account
3. delete account in settings
4. try to login again with account => this should fail

# Notes for the reviewer (optional)

# Notes for release (optional)
